assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -1012,6 +1013,49 @@ export default Component.extend({
     next(this.reStickScrollIfNeeded.bind(this));
   },
 
+  @discourseComputed("chatChannel.status")
+  channelStatusMessage(channelStatus) {
+    if (channelStatus === CHANNEL_STATUSES.open) {
+      return null;
+    }
+
+    switch (channelStatus) {
+      case CHANNEL_STATUSES.closed:
+        if (this.currentUser.staff) {
+          return I18n.t("chat.channel_status.closed_staff_header");
+        } else {
+          return I18n.t("chat.channel_status.closed_header");
+        }
+        break;
+      case CHANNEL_STATUSES.readOnly:
```

Comment:
Okay this is silly but I was staring hard at `readOnly` and I thought it should be [readonly](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly).

You don't have to change this, but it just gives me the funnies.

Reply:
But then maybe people would mix it up with the special attribute :P I know in rails I had to call it read_only because readonly had special meaning on the models